scripts/nightly-cleanup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+az login --service-principal --username $AZ_USERNAME --password $AZ_PASSWORD --tenant $AZ_TENANT
+repo_list=$(az acr repository list --name unstablebrigade)
+
+# remove leading and trailing []
+repo_list=${repo_list::-1}
+clean_repo_list=${repo_list:1}
```

Comment:
`repo_list=$(az acr repository list --name unstablebrigade -o table | tail -n +3)`

Would give you a clean list right out of the gate with no further need to massage the results before/during iterating the list.